<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UseVesselsInBlendingDialog</class>
 <widget class="QDialog" name="UseVesselsInBlendingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select the vessels participating in blend</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="chooseAllButton">
       <property name="toolTip">
        <string>Select all vessels</string>
       </property>
       <property name="text">
        <string>Choose all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseNoneButton">
       <property name="toolTip">
        <string>Deselect all vessels</string>
       </property>
       <property name="text">
        <string>Choose none</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="vesselsListWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chooseAllButton</tabstop>
  <tabstop>chooseNoneButton</tabstop>
  <tabstop>vesselsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UseVesselsInBlendingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UseVesselsInBlendingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
